<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>421</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_iniciar">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>140</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Iniciar</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>120</y>
      <width>411</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>200</y>
      <width>411</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Datos del semáforo activo</string>
    </property>
    <widget class="QLabel" name="semaforoActivo">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>91</width>
       <height>191</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Número de semáforo:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>140</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 11pt &quot;Arial&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>440</y>
      <width>411</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(157, 157, 157);</string>
    </property>
    <property name="title">
     <string>Prioridad</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>391</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="prioridades"/>
    </widget>
   </widget>
   <widget class="QPushButton" name="boton_apagar">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>140</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Apagar</string>
    </property>
   </widget>
   <widget class="QLabel" name="fondo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>551</width>
      <height>491</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../controlled-crossroads.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="semaforo1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="semaforo2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>280</y>
      <width>81</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="semaforo3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>270</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="semaforo4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>81</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>989</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
